manifests/gmp-operator/1_setup.yaml

```diff
@@ -1,3524 +0,0 @@
-# Copyright 2022 Google LLC
```

Comment:
I wonder if as a follow up its useful to have a manifest that does use the operator. Could be a useful way to see how operator versions affect the benchmarks over time (say if the operator adds some relabelling that proves to be very expensive).

Reply:
Yea, using operator blocks the architecture of this benchmark, but I see your point. It's actually quite easy to write a tool that will regenerate this scrape config based on operator code in prometheus-engine. Then we can hook it in CI/script, WDYT? Will add an issue

Reply:
https://github.com/bwplotka/prombenchy/issues/13